plugins/destination/file/client/spec.go

```diff
@@ -0,0 +1,34 @@
+package client
+
+import (
+	"fmt"
+)
+
+type FormatType string
+
+type BackendType string
+
+const (
+	FormatTypeCSV  = "csv"
+	FormatTypeJSON = "json"
+)
+
+type Spec struct {
+	Directory string     `json:"directory,omitempty"`
+	Format    FormatType `json:"format,omitempty"`
+	// This is used for debugging purposes only
+	NoRotate bool `json:"no_rotate,omitempty"`
```

Comment:
Should it be removed, if it's only for debugging? What does it do?

Reply:
It is also for user usage. Added documentation under `website`

Reply:
We should update the comment above the line then, it currently says that it's `for debugging purposes only`